index.d.ts

```diff
@@ -45,6 +45,12 @@ export interface constructorOptions {
  * Represents the callback in the APIs
  */
 export type apiCallback = () => void;
+
+/**
+ * Represents the callback in persistence queue
+ */
+export type persistenceQueueCallback = (error?: Error | string) => void;
```

Comment:
It's too narrow a use-case to have a dedicated type. Let's move this to the method declaration.

Reply:
Addressed